interview-questions.md

```diff
@@ -8,36 +8,82 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. As a developer, I am creating a Rails application with a model called Cohort and a model called Student. The cohort model has_many students, but OOPS! I forgot to add the foreign key. How can I fix this mistake? What is the name of the foreign key? Would the foreign key be on the Cohort model or the Student model?
 
-Your answer:
+Your answer: Add the update information through migrate.
 
 Researched answer:
 
+1. Go to terminal
+
+2. Generate a new migration
+
+3. Open the migration file and locate the change method
+
+4. Save the file
+
+5. Migrate through the command "rails db:migrate
+
+foreign key name: cohort_id
+
+The cohort model.
+
+
+
 2. Which RESTful routes must always be passed params? Why?
 
 Your answer:
 
-Researched answer:
+I know I've seen params id with show, I dont know the others. To identify the route.
+
+Researched answer: 
+
+The Show, Update, and Delete routes. They need to be identified specifically. Without parameters, it wouldn't know what to show, update or delete.
 
 3. Name three rails generator commands. What is created by each?
 
-Your answer:
+Your answer: 
 
-Researched answer:
+Model, Migrate, and cant remember the third. To take the workload.
+
+Researched answer: 
+
+Model, Controller, and Scaffold. They create the necessary files and boilerplate code to reduce the amount of manual work.
```

Comment:
This answer doesn't display an understanding of what each will generate. I would elaborate more.